pkg/util/constants.go

```diff
@@ -141,3 +141,11 @@ const (
 	// CacheSyncTimeout refers to the time limit set on waiting for cache to sync
 	CacheSyncTimeout = 30 * time.Second
 )
+
+const (
+	// Kubeconfig indicates the flag name of the kubeconfig
+	Kubeconfig = "kubeconfig"
+
+	// KubeconfigUsage show the usage of kubeconfig flag
+	KubeconfigUsage = "Path to karmada control plane kubeconfig file."
+)
```

Comment:
/assign @lonelyCZ 

Just a feeling that these const variables don't so `common`. 

In addition, we might be going to add more shared flags in the future(see #1321), I'm thinking if we need a common package to manage the shared flags.(like the [sharedcommand](https://github.com/karmada-io/karmada/tree/master/pkg/version/sharedcommand)).

Reply:
The meaning of `kubeconfig` is similar in components of Karmada. I think adding a command package of  shared flags is ok.

Reply:
Do any package and file names recommend?

Reply:
Like this?

https://github.com/karmada-io/karmada/blob/ca3bc35f685991d6604e2af10210d49fbf40bdcc/pkg/karmadactl/options/global.go#L16-L22

`karmada/pkg/options/sharedflag.go`?

Reply:
Prefer `karmada/pkg/options/sharedflag.go`.

How do you think @RainbowMango.

Reply:
package name `options` is a little bit too common that we have to use an alias. 

No better idea yet.

Reply:
How about moving it to `cmd/sharedconstant/sharedflags.go` file.